mlrun/model_monitoring/tracking_policy.py

```diff
@@ -1,111 +0,0 @@
-# Copyright 2023 Iguazio
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#   http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-#
-
-from typing import Union
-
-import mlrun.common.schemas.schedule
-import mlrun.model
-
-
-class TrackingPolicy(mlrun.model.ModelObj):
```

Comment:
I think here we need to deprecate it and to delete, maybe instead of log you can raise exception and explain that this class is irrelevant, and can't be used.